Windows/ppsspp.rc

```diff
@@ -68,6 +68,7 @@ BEGIN
     "W",            ID_EMULATION_STOP,      VIRTKEY, CONTROL, NOINVERT
     "B",            ID_EMULATION_RESET,     VIRTKEY, CONTROL, NOINVERT
     "T",            ID_EMULATION_CHEATS,    VIRTKEY, CONTROL, NOINVERT
+	"C",            ID_EMULATION_CHAT,      VIRTKEY, CONTROL, NOINVERT

```

Comment:
i add this hotkey by using visual studio view code. how to disbable that editorconfig or i must edit this code from another editor like notepad++ ? or add it from resource view on visual studio?